src/popover/popover.ts

```diff
@@ -140,6 +167,7 @@ export class NgbPopover implements OnInit, OnDestroy, OnChanges {
    */
   @Output() hidden = new EventEmitter();
 
+  private _autoCloseIgnoredElements = new Set<HTMLElement>();
```

Comment:
I don't think you ever put something in here and I would remove this from current PR for later if necessary

Reply:
Forgotten leftover, I'll remove